cmd/main.go

```diff
@@ -1,144 +1,30 @@
 package main
 
 import (
-	"flag"
-	"path/filepath"
-
-	log "github.com/sirupsen/logrus"
+	"log"
+	"os"
 
+	"github.com/openshift-kni/commatrix/cmd/generate"
 	"github.com/openshift-kni/commatrix/pkg/client"
-	commatrixcreator "github.com/openshift-kni/commatrix/pkg/commatrix-creator"
-	"github.com/openshift-kni/commatrix/pkg/consts"
-	"github.com/openshift-kni/commatrix/pkg/endpointslices"
-	listeningsockets "github.com/openshift-kni/commatrix/pkg/listening-sockets"
-	matrixdiff "github.com/openshift-kni/commatrix/pkg/matrix-diff"
-	"github.com/openshift-kni/commatrix/pkg/types"
-	"github.com/openshift-kni/commatrix/pkg/utils"
-)
 
-var (
-	destDir             string
-	format              string
-	customEntriesPath   string
-	customEntriesFormat string
-	debug               bool
-)
+	"github.com/spf13/pflag"
 
-func init() {
-	flag.StringVar(&destDir, "destDir", "communication-matrix", "Output files dir")
-	flag.StringVar(&format, "format", "csv", "Desired format (json,yaml,csv,nft)")
-	flag.StringVar(&customEntriesPath, "customEntriesPath", "", "Add custom entries from a file to the matrix")
-	flag.StringVar(&customEntriesFormat, "customEntriesFormat", "", "Set the format of the custom entries file (json,yaml,csv)")
-	flag.BoolVar(&debug, "debug", false, "Debug logs")
-	flag.Parse()
-}
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
 
 func main() {
-	if debug {
-		log.SetLevel(log.DebugLevel)
-	}
-
-	if customEntriesPath != "" && customEntriesFormat == "" {
-		log.Panic("Error: variable customEntriesFormat is not set")
-	}
+	flags := pflag.NewFlagSet("kubectl-commatrix", pflag.ExitOnError)
```

Comment:
What is this flag? The command is `kubectl commatrix` right?

Reply:
removed it 
not needed saw other repos like kube-compare and other not have like this flag